sdks/java/core/src/main/java/org/apache/beam/sdk/testing/BigqueryMatcher.java

```diff
@@ -121,8 +121,8 @@ protected boolean matchesSafely(PipelineResult pipelineResult) {
       throw new RuntimeException("Failed to fetch BigQuery data.", e);
```

Comment:
This block catch exceptions that throwed by `BackOffUtils.next`. It's different from query exceptions  so I handled it outside.